config.go

```diff
@@ -68,6 +70,12 @@ type Config struct {
 
 	// Consumer is the namespace for configuration related to consuming messages, used by the Consumer.
 	Consumer struct {
+		Retry struct {
+			// How long to wait after a partition has lost its leader before trying again (default 10s).
+			// Similar to the `refresh.leader.backoff.ms` setting of the JVM producer.
```

Comment:
kafka consumer seems to have both `refresh.leader.backoff.ms` and `rebalance.backoff.ms`